src/lib/systemd_journal/entries_dialog.rb

```diff
@@ -198,7 +198,7 @@ def read_search
 
     # Reads the journal entries from the system
     def read_journal_entries
-      log.info "Calling journalctl with '#{@query.journalctl_args}'"
+      log.info "Calling journalctl with #{@query.journalctl_options} and #{@query.journalctl_matches}"
```

Comment:
Well, right now ```@query.to_s``` is ```"<interval: #{@interval}, filters: #{@filters}>"``` which I actually find more reasonable (it's already used in other part of the code).